application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -634,33 +634,45 @@ define('diagram-store', ['jquery', 'xwiki-meta', 'xwiki-utils', 'diagram-utils',
     }
   };
 
+  var getTagName = function(element) {
+    if (!element) { return ''; }
+    return element.tagName.toLowerCase();
+  };
+
+  var isTextElement = function(element) {
```

Comment:
```suggestion
  var isTextNode = function(element) {
```

"TextElement" doesn't really make sense. A node is either a text node or an element node, but not both.